### ISSUE TITLE
Linux Modmerge does not work

### ISSUE THREAD

**Issue by azrael4h** [ScottBrooks/modmerge#2]
Linux Modmerge doe not actually do anything. It cannot be ran. After setting the executable bit, nothing happens when attempting to run.

**Comment by SomeTroglodyte** [comment]
Not true, runs just fine on Ubuntu 20.04. OP probably unaware it's a console app, no GUI.

**Comment by inhaledesign** [comment]
Confirming that it runs fine on Ubuntu 22.10.
